lib/services/access_list.go

```diff
@@ -140,6 +140,10 @@ type AccessListMemberGetter interface {
 	// implicit member list and the underlying implementation does not have
 	// enough information to compute the dynamic member record.
 	GetAccessListMember(ctx context.Context, accessList string, memberName string) (*accesslist.AccessListMember, error)
+	// GetAccessList returns the specified access list resource.
+	GetAccessList(context.Context, string) (*accesslist.AccessList, error)
+	// GetAccessList returns the specified access list resource.
+	GetAccessLists(context.Context) ([]*accesslist.AccessList, error)
```

Comment:
The interfaces name is `AccessListMemberGetter` 

By adding `GetAccessList` and `GetAccessLists` the interfaces name is obsolete.